src/tiny_websockets/network/generic_esp/generic_esp_clients.hpp

```diff
@@ -51,7 +54,9 @@ namespace websockets { namespace network {
       WSString line = "";
 
       int ch = -1;
+      uint64_t _timer = millis();
```

Comment:
No need for the leading underscore (see other variable names..)

Also consider using a more indicative name, like `millisBeforeReadingHeaders`